modules/ximgproc/src/deriche_filter.cpp

```diff
@@ -0,0 +1,410 @@
+#include "precomp.hpp"
```

Comment:
Please resolve this item too.
Also please check other license headers in your patch for missing space after `*` symbol:
```
*Redistributions of source code 
```